app/src/main/java/com/kickstarter/services/ApiClient.java

```diff
@@ -386,17 +387,16 @@ public ApiClient(final @NonNull ApiService service, final @NonNull Gson gson) {
   }
 
   @Override
-  public @NonNull Observable<Message> sendMessageToBacking(final @NonNull Backing backing, final @NonNull String body) {
-    return service
-      .sendMessageToBacking(backing.projectId(), backing.backerId(), MessageBody.builder().body(body).build())
-      .lift(apiErrorOperator())
-      .subscribeOn(Schedulers.io());
-  }
-
-  @Override
-  public @NonNull Observable<Message> sendMessageToThread(final @NonNull MessageThread messageThread, final @NonNull String body) {
-    return service
-      .sendMessageToThread(messageThread.id(), MessageBody.builder().body(body).build())
+  public @NonNull Observable<Message> sendMessage(final @NonNull MessageSubject messageSubject, final @NonNull String body) {
+    return messageSubject
+      .value(
+        backing -> this.service
+          .sendMessageToBacking(backing.projectId(), backing.backerId(), MessageBody.builder().body(body).build()),
```

Comment:
Can `MessageBody.builder().body(body).build()` go above to make these three cases shorter?